tutorials/custom-migration-tutorial.md

```diff
@@ -0,0 +1,109 @@
+# Upgrading to v1.1.1 using a Custom NamedKey Convention
+
+This tutorial uses the Casper command-line client to upgrade *and* migrate from an NFT contract installed using release [v1.0.0](https://github.com/casper-ecosystem/cep-78-enhanced-nft/releases/tag/v1.0.0) and customized NamedKeys. The outcome of the upgrade will be a [v1.1.1](https://github.com/casper-ecosystem/cep-78-enhanced-nft/releases/tag/v1.1.1) NFT contract with its storage structure migrated to the new format.
+
+## Prerequisites
+
+- You have previously installed a CEP-78 contract using release v1.0.0 on a Casper network. Thus, you have experience with: the [Casper CEP-78 NFT Standard](https://github.com/casper-ecosystem/cep-78-enhanced-nft/), the Casper command-line client, and interacting with a Casper network.
```

Comment:
`with:` reads a bit strange

Reply:
Thanks for catching this, I used to have bullets. I removed the `:` .

Reply:
Thanks, I have removed the `:`. It used to be a list.